src/components/ui/Footer.vue

```diff
@@ -1,43 +1,67 @@
 <template>
   <footer class="footer">
-    <a
-      v-for="(single, index) in links"
-      v-bind:key="index"
-      v-bind:href="single.link"
-      class="footer__link"
-      target="_blank"
-      rel="noopener noreferrer"
-    >
-      <Icon v-bind:id="single.icon" :width="12" :height="12" />
-      <abbr v-if="single.abbr" v-bind:title="single.abbr">
-        {{ single.label }}
-      </abbr>
-      {{ !single.abbr ? single.label : '' }}
-    </a>
-    <select v-if="languages.length > 1" v-on:change="updateLocale">
-      <option
-        v-for="[locale, language] in languages"
-        :key="locale"
-        :value="locale"
-        :selected="locale === currentLocale ? true : false"
-        >{{ language }}</option
+    <section class="footer__contribute" v-if="showContribute">
+      <h1>{{ fluent('contribute_header') }}</h1>
+      <div class="footer__contribute-img">
+        <img
+          src="@/assets/images/crowd.png"
+          srcset="@/assets/images/crowd@2x.png 2x"
+        />
+      </div>
+      <p>{{ fluent('contribute_text') }}</p>
+      <ExternalButtonLink
+        class="footer__contribute__link button--invert"
+        href="https://www.mozilla.org/contribute/"
+        iconRight="chevron-right"
+        :text="fluent('contribute')"
       >
-    </select>
+      </ExternalButtonLink>
+    </section>
+    <section class="footer__links">
+      <a
+        v-for="(single, index) in links"
+        v-bind:key="index"
+        v-bind:href="single.link"
+        class="footer__link"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Icon v-bind:id="single.icon" :width="12" :height="12" />
+        <abbr v-if="single.abbr" v-bind:title="single.abbr">
+          {{ single.label }}
+        </abbr>
+        {{ !single.abbr ? single.label : '' }}
+      </a>
+      <select v-if="languages.length > 1" v-on:change="updateLocale">
+        <option
+          v-for="[locale, language] in languages"
+          :key="locale"
+          :value="locale"
+          :selected="locale === currentLocale ? true : false"
+          >{{ language }}</option
+        >
+      </select>
+    </section>
   </footer>
 </template>
 
 <script>
 import Icon from '@/components/ui/Icon.vue';
 import LinksMixin from '@/components/_mixins/LinksMixin.vue';
 import Fluent from '@/assets/js/fluent';
+import ExternalButtonLink from '@/components/ui/ExternalButtonLink.vue';
 
 export default {
   name: 'Footer',
   mixins: [LinksMixin],
   components: {
     Icon,
+    ExternalButtonLink,
   },
   computed: {
+    showContribute() {
+      return !this.$store.state.scope.isLoggedIn && this.$route.name === 'Home';
```

Comment:
Should we put this "Home" string in a static variable?

Reply:
We have that in a couple of places. I'd do that in a follow up as it's a general refactoring and then we should do that with all the old routes.